.travis.yml

```diff
@@ -0,0 +1,9 @@
+script:
+- |
+  docker build --target test --tag todo-app:test .
+  docker run todo-app:test tests/test_view_model.py
+  docker run todo-app:test tests/test_integration.py
+  docker run --env TRELLO_TOKEN=$TRELLO_TOKEN \
+             --env TRELLO_KEY=$TRELLO_KEY \
+             --env TRELLO_BOARD_NAME=MyTodos \
+             todo-app:test tests/test_selenium.py
```

Comment:
Would be more scalable to split them up by folder rather than specifying files, but this is fine given how few test files there are.